apiserver/deploy/base/apiserver.yaml

```diff
@@ -84,6 +84,7 @@ rules:
   resources:
   - rayclusters
   - rayjobs
+  - rayservices
```

Comment:
Do they need to be synchronized? If so, we can open an issue to add the RABC consistency between `deploy/base/apiserver.yaml` and  `helm-chart/kuberay-apiserver/templates/role.yaml`.